tools/lsp/ui/resizer.slint

```diff
@@ -31,6 +33,8 @@ export component Resizer {
     property <length> handle-size: ResizeState.handle-size;
 
     resize-area := Rectangle {
+        border-width: 1px;
+        border-color: Palette.border;
```

Comment:
@ogoffart what is the reason to do so? This is how it looks like now with the gap?

<img width="243" alt="image" src="https://github.com/slint-ui/slint/assets/6715107/210096fa-2439-48d5-8a55-00f756a9cbc1">

Reply:
what if the content also had a 1px border? That wouldn't be visible since it is covered by the extra border.  Also how is this interacting with the blue selection rectangle?

Reply:
Ok yes make sense. Thank you.

Reply:
Done.